packages/apm/src/integrations/tracing.ts

```diff
@@ -352,6 +519,32 @@ export class Tracing implements Integration {
     }
   }
 
+  /**
+   * Adds offset to the span
+   *
+   * @param measureName name of the performance measure
+   * @param span Span to add data.offset to
+   */
+  private static _addOffsetToSpan(measureName: string, span: SpanClass): void {
+    if (global.performance) {
+      const name = `#sentry-${measureName}`;
+      performance.measure(name);
+      const measure = performance.getEntriesByName(name).pop();
```

Comment:
There can be more than one entry with the same name!

In this very page here, open the developer console and run:

```javascript
> performance.getEntriesByName("https://github.com/getsentry/sentry-javascript/pull/2474/review_comment/create")
(8) [PerformanceResourceTiming, PerformanceResourceTiming, PerformanceResourceTiming, PerformanceResourceTiming, PerformanceResourceTiming, PerformanceResourceTiming, PerformanceResourceTiming, PerformanceResourceTiming]
```

Reply:
That's true, but since this mark /measure is only so short, it only lives to calculate the offset.
Therefore I think this can never return more than one.